applications/bci_visualization/operators/reconstruction/convert_to_voxels_operator.py

```
@@ -0,0 +1,193 @@
+    while ctr < 100:
+        ctr += 1
+        inds = rng.choice(np.arange(len(ijk)), size=n, replace=False)
+        ins = ijk[np.array(inds), :]  # <- points
+        out = xyz[np.array(inds), :]  # <- mapped to
+        B = np.vstack([np.transpose(ins), np.ones(n, dtype=np.float32)])
+        if np.linalg.det(B) == 0:
+            continue
+    if np.linalg.det(B) == 0:
+        raise RuntimeError("Cannot compute affine, algorithm failed after 100 attempts")
+    D = 1.0 / np.linalg.det(B)
+    entry = lambda r, d: np.linalg.det(np.delete(np.vstack([r, B]), (d + 1), axis=0))
```

Ruff (0.14.10)

Do not assign a `lambda` expression, use a `def`

Rewrite `entry` as a `def`

(E731)